integration-test/src/it/setup/depMgmt1/Sample.groovy

```diff
@@ -0,0 +1,56 @@
+
+import groovy.json.JsonOutput
+import groovy.json.JsonSlurper
+import groovy.util.logging.Slf4j
+
```

Comment:
We're using this code to post-process the json files to remove the resolved lines.